packages/aragon-cli/src/cli.js

```diff
@@ -35,7 +35,6 @@ cmd.middleware(MIDDLEWARES)
 
 cmd.alias('env', 'environment')
 cmd.alias('h', 'help')
-cmd.alias('v', 'version')
```

Comment:
Do we have a good reason to remove this?

Reply:
Oh this cause what we discuss here: https://github.com/aragon/aragen/pull/79#discussion_r332967208

Reply:
Ah ok I wasn't aware of the issue between `aragon -v` and `aragon devchain -v`, thanks 🙂 
However `aragon devchain -v` doesn't seem to be working either, is this normal?

Reply:
Oh sorry about that, it was cause I needed to update aragen first. I did that yesterday